kin-sdk-core/src/androidTest/java/kin/sdk/core/KinAccountTest.java

```diff
@@ -21,13 +24,39 @@
     private final String PASSPHRASE = "testPassphrase";
     private final String TO_ADDRESS = "0x82CdC15705CE9f4565DDa07d78c92ff3d2717854";
 
+    /**
+     * First new account with 0 TOKEN and 0 ETH.
+     */
     private KinAccount kinAccount;
 
+    /**
+     * Imported account via private ECDSA Key, from testConfig.json
+     * The first account (importedAccounts.get(0)) will have 1000 TOKEN and 100 ETH, and can sendTransactions.
+     * All other accounts (1-9) will have only 100 ETH.
+     */
+    private List<KinAccount> importedAccounts = new ArrayList<>(10);
+
     @Override
     @Before
     public void setUp() throws Exception {
         super.setUp();
         kinAccount = kinClient.createAccount(PASSPHRASE);
+        importAccounts();
+    }
+
+    /**
+     * Import all accounts from {@link Config}.
+     */
+    private void importAccounts() {
+        List<PrivateKey> accounts = config.getAccounts();
+        for (int i = 0; i < accounts.size(); i++) {
```

Comment:
1. why not use `for (PrivateKey account: accounts)`
2. how about renaming PrivateKey class to Account ?

Reply:
1,2 👍